force-app/main/default/sharingRules/Account.sharingRules-meta.xml

```diff
@@ -1,32 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <SharingRules xmlns="http://soap.sforce.com/2006/04/metadata">
-    <sharingCriteriaRules>
-        <fullName>Multi_Market_Account_Visibility</fullName>
-        <accessLevel>Read</accessLevel>
-        <accountSettings>
-            <caseAccessLevel>None</caseAccessLevel>
-            <contactAccessLevel>Read</contactAccessLevel>
-            <opportunityAccessLevel>None</opportunityAccessLevel>
-        </accountSettings>
-        <description>Grants account read access to Global Team for Multi Market accounts</description>
-        <label>Multi-Market Account Visibility</label>
-        <sharedTo>
-            <roleAndSubordinatesInternal>Global_Team</roleAndSubordinatesInternal>
-        </sharedTo>
-        <criteriaItems>
-            <field>Market_Reach__c</field>
-            <operation>equals</operation>
-            <value>Multi Market</value>
-        </criteriaItems>
-        <includeRecordsOwnedByAll>true</includeRecordsOwnedByAll>
-    </sharingCriteriaRules>
     <sharingOwnerRules>
         <fullName>ANZ_Account_Visibility</fullName>
-        <accessLevel>Edit</accessLevel>
+        <accessLevel>Read</accessLevel>
         <accountSettings>
```

Comment:
@rthomp-prosci -- we still want the market level account sharing to be set to Edit. There were some updates made after this story was initially developed. that maybe didn't make it back to your org.